src/components/LogRow/RowRenderer/index.tsx

```diff
@@ -49,7 +50,15 @@ const ParsleyRow: RowRendererFunction = ({ logType, processedLogLines }) => {
         />
       );
     }
-    const Row = rowRendererMap[logType];
+
+    // At this point, logMetadata.renderingType is guaranteed to be defined from LogView/LoadingPage.tsx.
+    // logType can be removed from this calculation when the Parsley resmoke route is fully deprecated.
+    const Row =
+      logMetadata?.renderingType === "resmoke" ||
```

Comment:
I also don't quite understand this bit of logic. Is there a way we can preserve rowRenderMap it seems far more flexible and would allow the expansion of future row types in the future.

Reply:
I think this logic doesn't make sense because it doesn't do a good job at holistically bringing in `renderingType` to Parsley.  The logic that you reviewed utilizes parsley route and renderingType only for logs. I just pushed some code to decouple the route from renderingType, utilizes testlog renderingType from the GQL endpoint but also extrapolates rendering type per logType.